### DISCUSSION TITLE
FGBS222 - S2 Authenticated - discarded temperature reported

### DISCUSSION THREAD

**Discussion by candrea77** [zwave-js/zwave-js#6446] [Request Support / Investigate Issue] [answered]
Hi,
I've purchased a FGBS222 and included it using "**S2 - Authenticated**"

Unfortunately, it seems to report temperature reading not using S2 - Authenticated.
Is there a way to undesrtand if it report the data using "S0-Legacy" or completely without encryption (I would like to submit a bug report to Fibaro but not sure how it report data) ?

```
2023-10-22T16:51:03.672Z SERIAL « 0x0112000400ab0a600d09003105012200b7e0006c                          (20 bytes)
2023-10-22T16:51:03.673Z SERIAL » [ACK]                                                                   (0x06)
2023-10-22T16:51:03.673Z CNTRLR   [Node 171] command was received at a lower security level than expected - disc
                                  arding it...
2023-10-22T16:51:03.673Z DRIVER « [Node 171] [REQ] [ApplicationCommand]                              [discarded]
                                  └─[MultiChannelCCCommandEncapsulation]
                                    │ source:      9
                                    │ destination: 0
                                    └─[MultilevelSensorCCReport]
                                        type:  Air temperature
                                        scale: Celsius
                                        value: 18.3
```

**Comment by AlCalzone** [comment, marked as answer]
The Z-Wave specs explicitly forbid accepting non-encrypted commands when a device is included securely, which is why this report is discarded.

This report is using no encryption at all. If it were using S0, you'd see `SecurityCCCommandEncapsulation` in the log. If it were using S2 with a lower security class than intended, you'd also see that mentioned.

**Comment by candrea77** [comment]
> `The Z-Wave specs explicitly forbid accepting non-encrypted commands when a device is included securely, which is why this report is discarded.`

I know..... it was me, some times ago, asking for this feature 😁

> `This report is using no encryption at all. If it were using S0, you'd see SecurityCCCommandEncapsulation in the log. If it were using S2 with a lower security class than intended, you'd also see that mentioned.`

This is what I was looking for.


Anyway .... after a full reinterview .... now this device have started talking in S2 authenticated.
Sometimes strange things happen🤔🤔🤔

> _2023-10-23T18:24:48.520Z DRIVER « [Node 171] [REQ] [ApplicationCommand]
>                                   └─[Security2CCMessageEncapsulation]
>                                     │ sequence number: 78
>                                     │ security class:  S2_Authenticated
>                                     └─[MultiChannelCCCommandEncapsulation]
>                                       │ source:      10
>                                       │ destination: 0
>                                       └─[MultilevelSensorCCReport]
>                                           type:  Air temperature
>                                           scale: Celsius
>                                           value: 18.8_
